### ISSUE TITLE
嗯 returns no result

### ISSUE THREAD

**Issue by hiromichinomata** [flyerhzm/chinese_pinyin#27]
## Description
嗯 returns no result
嗯 is HSK6 word.
In Japanese Wiktionary, the pinyin is described as en1.
https://ja.wiktionary.org/wiki/嗯

## Environment
OS: macOS Mojave ver10.14.6
ruby: ruby 2.6.3p62
chinese_pinyin (1.0.1)

## Test Code

```
require 'chinese_pinyin'

puts Pinyin.t('嗯', tonemarks: true)

# =>
(no result)

```

**Comment by flyerhzm** [comment]
@ hiromichinomata fixed in version `1.0.2`

**Comment by hiromichinomata** [comment]
Thank you for quick fix.
